frontend/console/src/features/timeline/TimelineCronScheduled.tsx

```diff
@@ -0,0 +1,17 @@
+import type { CronScheduledEvent } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+import { formatTimestampShort } from '../../utils/date.utils.ts'
+import { verbRefString } from '../verbs/verb.utils'
+
+export const TimelineCronScheduled = ({ cron }: { cron: CronScheduledEvent }) => {
+  const title = `Cron (${cron.schedule}) scheduled at ${cron.scheduledAt}`
+  return (
+    <span title={title}>
+      {'Cron '}
+      <span className='text-indigo-500 dark:text-indigo-300'>{cron.schedule} </span>
```

Comment:
Since you have the same className defined on each of these 3 child spans, you could bubble that up to the wrapper span instead. i.e. `<span title={title}  className='text-indigo-500 dark:text-indigo-300'>` and no `className` attr below. Helps reduce duplication

Reply:
Thanks @deniseli. In this case, the schedule, verb ref, and time are colored differently from "Cron {} verb {} scheduled for", and the span would format all the text the same. I put a screenshot in the pr description.

Reply:
Ahhh gotcha! The screenshot is awesome, thanks for adding that :)